lib/pdk/validators/metadata_validator.rb

```diff
@@ -18,6 +18,11 @@ def self.metadata_validators
       def self.invoke(report, options = {})
         exit_code = 0
 
+        if options[:targets] && options[:targets] != []
+          PDK.logger.info(_('metadata validator only checks metadata.json. The specified files will be ignored: %{targets}') % { targets: options[:targets].join(', ') })
+          options.delete(:targets)
```

Comment:
does `options` get `dup`ed somewhere? otherwise this might destroy data for other callers.